src/test/java/com/puppycrawl/tools/checkstyle/checks/header/MultiFileRegexpHeaderCheckTest.java

```diff
@@ -346,8 +348,11 @@ public void testGetExternalResourceLocationsFromMultipleFilesSet() throws Except
             .that(locationsWithHeaders.size())
             .isEqualTo(2);
 
-        final String expectedUri4 = new File(header4Path).toURI().toASCIIString();
-        final String expectedUri1 = new File(header1Path).toURI().toASCIIString();
+        final Path header4PathObj = Paths.get(header4Path);
+        final Path header1PathObj = Paths.get(header1Path);
```

Comment:
Please inline this veriables to get string from file by single line.
You have space, and you have problems to name it, good signal it is not required